src/App/ComponentsDocumentation/components/Sheet/index.js

```diff
@@ -68,7 +68,7 @@ const ContentGuidelines = () => (
                     <img src={`${basename}img/documentation/sheet/content-guidelines.png`} alt="Sheet component with sticky header and footer" className="img-fluid"/>
                 </div>
             </div>
-            <ul>
+            <ul className="list list-bullet">
                 <li>The <b>header</b> should have a title and a close button.</li>
                 <li>The <b>main content</b> area can contain a number of different components and this area can be vertically scrollable.</li>
                 <li>The <b>footer</b> is optional and designed to contain buttons that will lead to certain actions. The footer is also sticky.</li>
```

Comment:
This ul is not in line with the rest of the doc.

![image](https://user-images.githubusercontent.com/44967773/153871260-75584e55-e9bc-4a23-a58e-4d715c60c8b1.png)